docs/_docs/clustering/discovery-in-the-cloud.adoc

```diff
@@ -268,3 +269,47 @@ include::{javaFile}[tag=google,indent=0]
 tab:C#/.NET[unsupported]
 tab:C++[unsupported]
 --
+
+== Azure Blob Storage
+
+Ignite supports automatic node discovery by utilizing Azure Blob Storage.
+This mechanism is implemented in `TcpDiscoveryAzureBlobStorageIpFinder`.
+On start-up, each node registers its IP address in the storage and discovers other nodes by reading the storage.
+
+IMPORTANT: To use `TcpDiscoveryAzureBlobStorageIpFinder`, enable the `ignite-azure` link:setup#enabling-modules[module] in your application.
+
+IMPORTANT: This functionality is currently supported only for Ipv4 IP addresses. Please ensure that you are using the
```

Comment:
I don't think we can ignore IPv6. Please fix the IPv6 scenario properly by applying URL encoding, or by stripping interface names explicitly from the addresses.